<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orthographic projection</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_model"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_programm"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_orthographic_projection">
       <property name="text">
        <string>Orthographic projection</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Camera coordinates</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="vp_x">
          <property name="text">
           <string>x = %0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vp_y">
          <property name="text">
           <string>y =%0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vp_z">
          <property name="text">
           <string>z = %0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Controls</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="wireframe">
          <property name="text">
           <string>Render as wireframe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="normals">
          <property name="text">
           <string>Display normals vectors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chooseModelColor">
          <property name="text">
           <string>Choose model color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chooseBgColor">
          <property name="text">
           <string>Choose background color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadModel">
          <property name="text">
           <string>Load model</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Model info</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="file">
          <property name="text">
           <string>File: %0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="points">
          <property name="text">
           <string>Points: %0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="edges">
          <property name="text">
           <string>Edges: %0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="faces">
          <property name="text">
           <string>Faces: %0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Load_model">
   <property name="text">
    <string>&amp;Load model</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionAbout_programm">
   <property name="text">
    <string>About programm</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadModel</sender>
   <signal>clicked()</signal>
   <receiver>action_Load_model</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
